contracts/implement/ConcentratedLiquidity.sol

```diff
@@ -90,10 +90,19 @@ abstract contract ConcentratedLiquidity {
         uint256 _amount
     ) internal virtual {}
 
+    // 1. Burn NFT
+    // 2. Update liquidity data
+    // 3. Transfer assets
     function removeLiquidity(uint256 tokenId) public virtual {
         address owner = _msgSender();
         require(owner == positionDexNft.ownerOf(tokenId), "!Owner");
         // TODO removeLiquidity and get fee reward
+
```

Comment:
call function `removeLiquidity` in the pool. Get amount baseOut and quoteOut and transfer